packages/playground-gist_backend/test.js

```diff
@@ -0,0 +1,35 @@
+/* check if the server starts without error message and then shut it down after 3s */
```

Comment:
Probably later but storing API key in Github and then actually trying to submit an example TD would be more desirable as a test?

Reply:
Yes I agree. We can add this later as you say. Most tests could be enhanced in the future.